CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+== Version 8.1.0
+
+* Release 2020-01 REST ADMIN API VERSION [#656](https://github.com/Shopify/shopify_api/pull/)
```

Comment:
Both pull request URLs are missing their PR ID numbers, so they currently point to 404s. URLs should be:
- https://github.com/Shopify/shopify_api/pull/656
- https://github.com/Shopify/shopify_api/pull/657